index.html

```diff
@@ -332,130 +323,134 @@
-        "https://heycam.github.io/webidl/#idl-ArrayBuffer">ArrayBuffer</a></dfn>,
-        <dfn><a href=
-        "https://heycam.github.io/webidl/#idl-ArrayBufferView">ArrayBufferView</a></dfn>
-        are defined in [[!WEBIDL]].
+        [[!WEBIDL-1]].
       </p>
       <p>
         The term <dfn><a href=
         "https://heycam.github.io/webidl/#dfn-throw">throw</a></dfn> in this
-        specification is used as defined in [[!WEBIDL]]. The following
-        exception names are defined by WebIDL and used by this specification:
+        specification is used as defined in [[!WEBIDL-1]].
+      </p>
+      <p>
+        The terms <dfn><a href=
+        "https://heycam.github.io/webidl/#idl-promise">Promise</a></dfn>,
+        <dfn><a href=
+        "https://heycam.github.io/webidl/#idl-ArrayBuffer">ArrayBuffer</a></dfn>,
+        <dfn><a href=
+        "https://heycam.github.io/webidl/#idl-ArrayBufferView">ArrayBufferView</a></dfn>,
+        and the following exception names are defined in [[!WEBIDL-1]]:
       </p>
       <ul>
         <li>
-          <dfn><code><a href=
-          "https://heycam.github.io/webidl/#notallowederror">NotAllowedError</a></code></dfn>
+          <a href=
+          "https://heycam.github.io/webidl/#invalidaccesserror"><dfn><code>InvalidAccessError</code></dfn></a>
         </li>
         <li>
-          <dfn><code><a href=
-          "https://heycam.github.io/webidl/#invalidaccesserror">InvalidAccessError</a></code></dfn>
+          <a href=
+          "https://heycam.github.io/webidl/#notfounderror"><dfn><code>NotFoundError</code></dfn></a>
         </li>
         <li>
-          <dfn><code><a href=
-          "https://heycam.github.io/webidl/#notfounderror">NotFoundError</a></code></dfn>
+          <a href=
+          "https://heycam.github.io/webidl/#notsupportederror"><dfn><code>NotSupportedError</code></dfn></a>
         </li>
         <li>
-          <dfn><code><a href=
-          "https://heycam.github.io/webidl/#notsupportederror">NotSupportedError</a></code></dfn>
+          <a href=
+          "https://heycam.github.io/webidl/#operationerror"><dfn><code>OperationError</code></dfn></a>
         </li>
         <li>
-          <dfn><code><a href=
-          "https://heycam.github.io/webidl/#operationerror">OperationError</a></code></dfn>
+          <a href=
+          "https://heycam.github.io/webidl/#securityerror"><dfn><code>SecurityError</code></dfn></a>
         </li>
         <li>
-          <dfn><code><a href=
-          "https://heycam.github.io/webidl/#securityerror">SecurityError</a></code></dfn>
+          <a href=
+          "https://heycam.github.io/webidl/#syntaxerror"><dfn><code>SyntaxError</code></dfn></a>
         </li>
         <li>
-          <dfn><code><a href=
-          "https://heycam.github.io/webidl/#syntaxerror">SyntaxError</a></code></dfn>
+          <a href=
+          "https://heycam.github.io/webidl/#notallowederror"><dfn><code>NotAllowedError</code></dfn></a>,
```

Comment:
Are the other exception names *not* defined in WEBIDL-2?  In other words, would it be possible to update all exception name references to WEBIDL-2?